backend/core/config.py

```
@@ -0,0 +1,199 @@
+class Settings(BaseSettings):
+    """Application settings"""
+
+    # Application settings
+    APP_NAME: str = "CloudOpsTools"
+    VERSION: str = "2.0.0"
+    ENVIRONMENT: str = "development"
+    DEBUG: bool = True
+    DEV_MODE: bool = False  # Set via environment variable, enables mock AWS operations
+
+    # Security
+    SECRET_KEY: str = "your-secret-key-here-change-in-production"
```

Ruff (0.14.10)

Possible hardcoded password assigned to: "SECRET_KEY"

(S105)